.gitmodules

```diff
@@ -202,6 +202,9 @@
 [submodule "projects/Neurocon"]
 	path = projects/Neurocon
 	url = https://github.com/conpdatasets/Neurocon.git
+[submodule "projects/_3_step_CPCA"]
+	path = projects/_3_step_CPCA
+	url = https://github.com/conp-bot/conp-dataset-3-step-CPCA.git
```

Comment:
This is an old unresolved submission from a UBC student and CONP Scholar named Abhijit Chinchani, for whom we did a research spotlight [here](https://conp.ca/research-spotlight-2/).  The problem is that it isn't so much a dataset as bit of code for his analysis technique.

After encouraging him to translate his code from Matlab to Python, I spoke to Abhijit about possibly converting his Jupyter notebook to a NeuroLibre book but I haven't had any new since then.  Will follow up with him.